src/renderers/shared/reconciler/__tests__/ReactUpdates-test.js

```diff
@@ -785,7 +787,6 @@ describe('ReactUpdates', function() {
         return <div />;
       },
       componentDidUpdate: function() {
-        var component = this;
```

Comment:
Yes. `component` was being shadowed (see line 802), but I both declarations held the same value. This part of the test is checking `asap` sets `this` correctly, which I believe this version still checks for.